src/Core/AdminConsole/Entities/Organization.cs

```diff
@@ -300,7 +313,6 @@ public void UpdateFromLicense(OrganizationLicense license)
         UseSecretsManager = license.UseSecretsManager;
         SmSeats = license.SmSeats;
         SmServiceAccounts = license.SmServiceAccounts;
-        LimitCollectionCreationDeletion = license.LimitCollectionCreationDeletion;
```

Comment:
This method is used when **updating** an existing org from a license file. You've removed the `LimitCollectionCreationDeletion` assignment, but not `AllowAdminAccessToAllCollectionItems`, which is inconsistent.

Also, the `OrganizationService.SignUpAsync` method (which **creates** a new org from a license file) has this changed feature flagged, but here it's not feature flagged.

Reply:
I'm also not sure about the value of feature flagging self-hosted code given that we can't dynamically control it. It ends up being a code change to turn the feature flag on anyway.

Reply:
I've added this removed property assignment back, and feature toggled both it and the `AllowAdminAccessToAllCollectionItems` assignment. This should look the same between `Create` and `Update` operations now.

__Commit:__ [Improve feature toggling scenerios for self hosted org creation/update](https://github.com/bitwarden/server/pull/4730/commits/aeeb796aed06565dfbba62a90d9676fcabef7eb7)

Reply:
This looks good to me now, it's all following the feature flag.